eng/Packages.Data.props

```diff
@@ -288,9 +289,11 @@
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.2" />
     <PackageReference Update="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Update="Microsoft.Extensions.Azure" Version="1.6.3" />
+	<PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="7.0.0" />
```

Comment:
Any reason that we shouldn't push these to the latest?   I believe the 8.0 line should work just fine with `net6.0` and `net7.0` targets.  Not particularly important nor blocking.   I can always bump them in a follow-up.

Reply:
@jsquire 7.0 would've been what I got when VS added the nuget package reference - I assume as it's targets net 7, it picked that one. I'm happy to bump and retest if you want me to?

Reply:
@davetrainer:  I appreciate your willingness, but no need.   I'll need to do a follow-up to uncomment the new sample link once it's live.   I'll play with the package references when I do that one.   I'm just waiting on one other thing to close internally and then I'll get this merged in.